tests/setup/mod.rs

```diff
@@ -163,16 +186,21 @@ pub mod nigiri {
 
         start_nigiri();
         start_lspd();
+        start_rgs();
     }
 
     pub fn stop() {
+        debug!("RGS server stopping ...");
+        stop_rgs();
         debug!("LSPD stopping ...");
         stop_lspd(); // Nigiri cannot be stopped if lspd is still connected to it.
         debug!("NIGIRI stopping ...");
         exec(&["nigiri", "stop", "--delete"]);
     }
 
     pub fn pause() {
+        debug!("RGS server stopping ...");
```

Comment:
would it make sense to move the debug log to the `stop_rgs` function, to keep it consistent with the `start_rgs` function?